@jules the failure seems to be:
```
2026-08-21T03:11:27.0838578Z  *  - INOTIFY_USER
2026-08-21T03:11:27.0838935Z  * You're on your own to make sure they are set if needed.
2026-08-21T03:11:27.0839128Z 
2026-08-21T03:11:27.0839244Z  * Messages for package media-libs/fontconfig-2.17.1:
2026-08-21T03:11:27.0839427Z 
2026-08-21T03:11:27.0839565Z  * Please make fontconfig configuration changes using `eselect
2026-08-21T03:11:27.0839899Z  * fontconfig`. Any changes made to /etc/fonts/fonts.conf will be
2026-08-21T03:11:27.0840242Z  * overwritten. If you need to reset your configuration to upstream
2026-08-21T03:11:27.0840587Z  * defaults, delete the directory /etc/fonts/conf.d/ and re-emerge
2026-08-21T03:11:27.0840859Z  * fontconfig.
2026-08-21T03:11:27.0841099Z  * (Note: Above message is only printed the first time package is
2026-08-21T03:11:27.0841459Z  * installed. Please look at /usr/share/doc/fontconfig-2.17.1/README.gentoo*
2026-08-21T03:11:27.0841771Z  * for future reference)
2026-08-21T03:11:27.0841897Z 
2026-08-21T03:11:27.0841999Z  * Messages for package x11-libs/gtk+-3.24.52:
2026-08-21T03:11:27.0842167Z 
2026-08-21T03:11:27.0842319Z  * Please install app-text/evince for print preview functionality.
2026-08-21T03:11:27.0842686Z  * Alternatively, check "gtk-print-preview-command" documentation and
2026-08-21T03:11:27.0842998Z  * add it to your settings.ini file.
2026-08-21T03:11:27.0860808Z 
2026-08-21T03:11:27.0860979Z  * GNU info directory index is up-to-date.
2026-08-21T03:11:27.1111730Z 
2026-08-21T03:11:27.1112332Z  * IMPORTANT: 19 news items need reading for repository 'gentoo'.
2026-08-21T03:11:27.1112715Z  * Use eselect news read to view new items.
2026-08-21T03:11:27.1112903Z 
2026-08-21T03:11:27.2554658Z + '[' -n ' =www-misc/which_browser-0.2.6.44-r1 (pkgcheck)' ']'
2026-08-21T03:11:27.2556002Z The following packages failed to build, install, or pass pkgcheck:  =www-misc/which_browser-0.2.6.44-r1 (pkgcheck)
2026-08-21T03:11:27.2557035Z + echo 'The following packages failed to build, install, or pass pkgcheck:  =www-misc/which_browser-0.2.6.44-r1 (pkgcheck)'
2026-08-21T03:11:27.2558088Z + exit 1
2026-08-21T03:11:27.2608056Z ##[error]Process completed with exit code 1.
2026-08-21T03:11:27.2649006Z ##[group]Run docker stop gentoo || true
2026-08-21T03:11:27.2649295Z [36;1mdocker stop gentoo || true[0m
2026-08-21T03:11:27.2649524Z [36;1mdocker stop portage || true[0m
2026-08-21T03:11:27.2649759Z [36;1mdocker rm --force gentoo || true[0m
2026-08-21T03:11:27.2650009Z [36;1mdocker rm --force portage || true[0m
2026-08-21T03:11:27.2671682Z shell: /usr/bin/bash -e {0}
2026-08-21T03:11:27.2671926Z ##[endgroup]
2026-08-21T03:11:37.5874398Z gentoo
2026-08-21T03:11:37.5990428Z portage
2026-08-21T03:11:38.5127651Z gentoo
2026-08-21T03:11:38.6265261Z portage
2026-08-21T03:11:38.6381042Z Node 20 is being deprecated. This workflow is running with Node 24 by default. If you need to temporarily use Node 20, you can set the ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION=true environment variable. For more information see: https://github.blog/changelog/2025-09-19-deprecation-of-node-20-on-github-actions-runners/
2026-08-21T03:11:38.6382413Z Post job cleanup.
2026-08-21T03:11:38.7127950Z [command]/usr/bin/git version
2026-08-21T03:11:38.7161449Z git version 2.55.0
2026-08-21T03:11:38.7188827Z Temporarily overriding HOME='/home/runner/work/_temp/f906b953-b13d-4a42-8352-36ec12e83f74' before making global git config changes
2026-08-21T03:11:38.7189955Z Adding repository directory to the temporary git global config as a safe directory
2026-08-21T03:11:38.7195116Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/arrans_overlay/arrans_overlay
2026-08-21T03:11:38.7222602Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2026-08-21T03:11:38.7249061Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2026-08-21T03:11:38.7422929Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2026-08-21T03:11:38.7442220Z http.https://github.com/.extraheader
2026-08-21T03:11:38.7450526Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2026-08-21T03:11:38.7475572Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2026-08-21T03:11:38.7663037Z [command]/usr/bin/git config --local --name-only --get-regexp ^includeIf\.gitdir:
2026-08-21T03:11:38.7691246Z [command]/usr/bin/git submodule foreach --recursive git config --local --show-origin --name-only --get-regexp remote.origin.url
2026-08-21T03:11:38.7980542Z Cleaning up orphan processes
2026-08-21T03:11:38.8205518Z ##[warning]Node.js 20 is deprecated. The following actions target Node.js 20 but are being forced to run on Node.js 24: actions/checkout@v4. For more information see: https://github.blog/changelog/2025-09-19-deprecation-of-node-20-on-github-actions-runners/
```

--- a/.github/workflows/gentoo-pkg-test.yml
+++ b/.github/workflows/gentoo-pkg-test.yml
@@ -107,7 +107,7 @@ jobs:
           FAILED_PACKAGES=""
           for PKG in $PACKAGES; do
             echo "Running pkgcheck on $PKG"
-            if ! pkgcheck scan --net --exit error /var/db/repos/arrans-overlay/${PKG#=}; then
+            if ! pkgcheck scan --net --exit error -R p "$PKG"; then
               echo "pkgcheck failed for $PKG"
               FAILED_PACKAGES="$FAILED_PACKAGES $PKG (pkgcheck)"
             fi

--- a/.github/workflows/gentoo-pkg-test.yml
+++ b/.github/workflows/gentoo-pkg-test.yml
@@ -107,7 +107,8 @@ jobs:
           FAILED_PACKAGES=""
           for PKG in $PACKAGES; do
             echo "Running pkgcheck on $PKG"
-            if ! pkgcheck scan --net --exit error -R p "$PKG"; then
+            CP=$(python3 -c "import portage; print(portage.dep.Atom(\"$PKG\").cp)")
+            if ! pkgcheck scan --net --exit error /var/db/repos/arrans-overlay/${CP}; then
               echo "pkgcheck failed for $PKG"
               FAILED_PACKAGES="$FAILED_PACKAGES $PKG (pkgcheck)"
             fi

--- a/.github/workflows/gentoo-pkg-test.yml
+++ b/.github/workflows/gentoo-pkg-test.yml
@@ -107,7 +107,7 @@ jobs:
           FAILED_PACKAGES=""
           for PKG in $PACKAGES; do
             echo "Running pkgcheck on $PKG"
-            if ! pkgcheck scan /var/db/repos/arrans-overlay/${PKG#=}; then
+            if ! pkgcheck scan --net --exit error /var/db/repos/arrans-overlay/${PKG#=}; then
               echo "pkgcheck failed for $PKG"
               FAILED_PACKAGES="$FAILED_PACKAGES $PKG (pkgcheck)"
             fi

